experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jenkins/jenkins:2.401.1-lts
+USER root
+RUN echo "2.0" > /usr/share/jenkins/ref/jenkins.install.UpgradeWizard.state
+COPY plugins.txt /usr/share/jenkins/ref/plugins.txt
+RUN jenkins-plugin-cli --plugin-file /usr/share/jenkins/ref/plugins.txt
+COPY jenkins.yaml /var/jenkins_home/casc_configs/jenkins.yaml
+ENV CASC_JENKINS_CONFIG /var/jenkins_home/casc_configs/jenkins.yaml
```

Comment:
Please set back the user to "jenkins" as mentioned [here](https://github.com/jenkinsci/docker#installing-more-tools)

Reply:
I've made the changes Please take a look